### PR TITLE
Update geolocation-!cn

### PR DESCRIPTION
**Pull request by Robot-DaneelOlivaw**
Add domains of a few services and forums. Update `geolocation-!cn` as well.

### CODE REVIEW

**Comment by EpLiar** [comment]
内容其他人在 review 了，然后就是 PR 的粒度尽可能的小一些。比如这个 PR 叫做 Update geolocation-!cn，然后你顺手改了下战旗和 geolocation-cn，这样会加大 PR 的复杂度。还是拆成两个 PR 会好一些

File: data/vk [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+vk-portal.net
+vk.cc
+vk.com
+vk.design
+vk.link
+vk.me
+vkcache.com
+vkgo.app
+vklive.app
+vkmessenger.com
```
Comment (MuenYu) [inline data/vk:13]: 域名访问超时，whois无法查询到公司归属，请问域名的来源是？

File: data/vk [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+vkmessenger.app
+vkontakte.ru
+vkuseraudio.com
+vkuserlive.net
```
Comment (MuenYu) [inline data/vk:17]: 域名访问404，whois无法查询到公司归属，请问域名的来源是？

File: data/espn [hunk 3]
```diff
@@ -0,0 +1,6 @@
+espn.com
+espncdn.com
+watchespn.com
+
+full:espn.hb.omtrdc.net
+full:espndotcom.tt.omtrdc.net
```
Comment (MuenYu) [inline data/espn:6]: 请问域名来源是？

File: data/vk (hunk 2 above)
Comment (KevinZonda) [reply data/vk:17]: Pay attention to the ASN of A records. Meanwhile, there are about 476 subdomains of it.

File: data/espn (hunk 3 above)
Comment (Robot-DaneelOlivaw) [reply data/espn:6]: [espn.com](https://www.espn.com)

File: data/vk (hunk 1 above)
Comment (Robot-DaneelOlivaw) [reply data/vk:13]: `mail.vkmessenger.com`未超时，其IP的自治系统编号属于VKontakte Ltd，因此鸭子测试其为VK旗下域名。

File: data/vk (hunk 2 above)
Comment (Robot-DaneelOlivaw) [reply data/vk:17]: 其IP的自治系统编号属于VKontakte Ltd，主机名称为`srv187-129-240-87.vk.com`，因此鸭子测试其为VK旗下域名。

File: data/vk (hunk 1 above)
Comment (KevinZonda) [reply data/vk:13]: Pay attention to the ASN of resolution records. Meanwhile, there are over 1000 subdomains of it.

File: data/vk (hunk 2 above)
Comment (KevinZonda) [reply data/vk:17]: What does 鸭子 mean? Ducksoft or?

Comment (Robot-DaneelOlivaw) [reply data/vk:17]: > What does 鸭子 mean? Ducksoft or?

If something looks, swims and quacks like a duck, it's probably a duck. A great methodology to get lazy.

File: data/espn (hunk 3 above)
Comment (MuenYu) [reply data/espn:6]: 不好意思，是浏览器插件给拦截了。

File: data/vk (hunk 2 above)
Comment (EpLiar) [reply data/vk:17]: > What does 鸭子 mean? Ducksoft or?

DuckSoft feels very cao

**Pull request merged**